netbox/dcim/api/serializers.py

```diff
@@ -1076,7 +1076,7 @@ def get_path(self, obj):
 
 class VirtualChassisSerializer(NetBoxModelSerializer):
     url = serializers.HyperlinkedIdentityField(view_name='dcim-api:virtualchassis-detail')
-    master = NestedDeviceSerializer(required=False)
+    master = NestedDeviceSerializer(required=False, allow_null=True)
```

Comment:
I think this also needs `default=None` as similar fields have defined.

```suggestion
    master = NestedDeviceSerializer(required=False, allow_null=True, default=None)
```